### PR TITLE
Hotfix/making blog site specific to ciaranmckenna

### PR DESCRIPTION
**Pull request by ciaranmckenna**

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -56,8 +56,7 @@ const Header = ({ siteTitle }) => (
       <h4>
         I'm a Software Engineer based in Belfast, N. Ireland.{' '}
         <span className="no-print">
-          This is my infrequent blog on technology, politics &amp; life
-          generally.
+          Here I occasionly discuss technology, politics &amp; healthy living.
```
Comment (barrymcgee) [inline src/components/header.js]:
```suggestion
          Here I occasionally discuss technology, politics &amp; healthy living.
```

**Review by barrymcgee — CHANGES_REQUESTED** [review]
Just found one small typo 👍🏻

**Comment by barrymcgee** [comment]
@ciaranmckenna Also note, conventation dictates;

- Branch names should be short and lowercase
- You have this branch in a 'hotfix' folder. a 'hotfix' is the term used for small unavoidable changes to fix a production bug outside of your regular development cycle.

File: src/components/header.js (hunk 1 above)
Comment (ciaranmckenna) [reply src/components/header.js]: Cool, thanks.

**Review by barrymcgee — APPROVED** [review]

**Comment by barrymcgee** [comment]
@ciaranmckenna This is now good to merge.

File: src/pages/about.js [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 const About = ({ location }) => (
   <Layout location={location}>
     <SEO title="About me" />
     <h1>About me</h1>
     <p>
-      I am a front end web developer with over ten years experience of building
-      websites used by millions of people. I care about crafting a web that
-      looks good, loads fast &amp; works everywhere, for everyone.
+      I am a recent software development graduate, beginning my career as a back end developer. I was previously
+      a teacher and head of department before joinging the world of software development.
```
Comment (ciaranmckenna) [inline src/pages/about.js]:
```suggestion
      a teacher and head of department before joining the world of software development.
```

**Comment by ciaranmckenna** [comment]
Reviewed in person with @barrymcgee

**Pull request merged**